### PR TITLE
Customize menu colors

### PR DESCRIPTION
**Pull request by loriepisicchio**
**Issue**

https://gravitee.atlassian.net/browse/APIM-3109

**Description**

Customize menu colors.

---
#### 🧪&nbsp;&nbsp;Gravitee.io Automatic Prerelease @gravitee/ui-particles-angular
```
npm install @gravitee/ui-particles-angular@7.38.0-menu-customization-976b00f
```
```
yarn add @gravitee/ui-particles-angular@7.38.0-menu-customization-976b00f
```

---
#### 🧪&nbsp;&nbsp;Gravitee.io Automatic Prerelease @gravitee/ui-policy-studio-angular
```
npm install @gravitee/ui-policy-studio-angular@7.38.0-menu-customization-976b00f
```
```
yarn add @gravitee/ui-policy-studio-angular@7.38.0-menu-customization-976b00f
```

---

### CODE REVIEW

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-footer/gio-menu-footer.component.scss [hunk 1]
```diff
@@ -17,13 +17,17 @@
 @use '@angular/material' as mat;
 @use '../../../scss' as gio;
 
+$background: map.get(gio.$mat-oem-palette, background);
+$text: map.get(gio.$mat-oem-palette, background-contrast);
+$menu-border: color-mix(in srgb, $background 50%, $text);
```
Comment (jourdiw) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-footer/gio-menu-footer.component.scss:22]: Only the `menu-border` is used in this file.

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-selector/gio-menu-selector.component.scss [hunk 2]
```diff
@@ -34,19 +36,49 @@ $text: map.get(gio.$mat-dove-palette, default);
   }
 }
 
+.gio-menu-selector__mat-option.mat-option.mat-selected {
+  color: #000000; // TODO: Change this to $accent or $text...
```
Comment (jourdiw) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-selector/gio-menu-selector.component.scss:40]: We should check with @audrey-tlm what she thinks... Before it was a the green primary color. The drop-down is also always with a white background, so no need to link it to $text I think....

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-selector/gio-menu-selector.component.scss [hunk 3]
```diff
@@ -34,19 +36,49 @@ $text: map.get(gio.$mat-dove-palette, default);
...
   gio-menu-selector {
     .gio-menu-selector {
       > * {
         display: block;
       }
 
-      @include mat.select-color(gio.$mat-theme-dark);
-      @include mat.form-field-color(gio.$mat-theme-dark);
-      @include mat.core-color(gio.$mat-theme-dark);
+      //@include mat.select-color(gio.$mat-theme-dark);
```
Comment (jourdiw) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-selector/gio-menu-selector.component.scss:50]: I think we can fully delete this. If I understand correctly, it's `::ng-deep` but only for when there is a `mat-selector` in `gio-menu-selector`...

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-submenu/gio-submenu-group/gio-submenu-group.component.scss [hunk 4]
```diff
@@ -18,8 +18,11 @@
 
 @use '../../../scss' as gio;
 
-$text: map.get(gio.$mat-space-palette, lighter60);
-$text-light: mat.get-color-from-palette(gio.$mat-dove-palette, darker20);
+$menuText: map.get(gio.$mat-oem-palette, background-contrast);
```
Comment (jourdiw) [inline ui-particles-angular/projects/ui-particles-angular/src/lib/gio-submenu/gio-submenu-group/gio-submenu-group.component.scss:21]: `menuText` and `menuTextLight` are not used here

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-footer/gio-menu-footer.component.scss (hunk 1 above)
Comment (loriepisicchio) [reply ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-footer/gio-menu-footer.component.scss:22]: background and text are use in the color-mix

File: ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-selector/gio-menu-selector.component.scss (hunk 2 above)
Comment (loriepisicchio) [reply ui-particles-angular/projects/ui-particles-angular/src/lib/gio-menu/gio-menu-selector/gio-menu-selector.component.scss]: I decided to hardcode it to black for now. Maybe we'll find a way to improve this later.

**Review by jourdiw — APPROVED** [review]
LGTM

**Pull request merged**